foundry/applications/formApplications/baseEntitySheets/journalSheet.d.ts

```diff
@@ -1,23 +1,67 @@
 /**
  * The JournalEntry Configuration Sheet
- *
- * @param entity - The JournalEntry instance which is being edited
- * @param options - Application options
  */
-declare class JournalSheet extends BaseEntitySheet {
+declare class JournalSheet<
+  D extends object = BaseEntitySheet.Data<JournalEntry>,
+  O extends JournalEntry = D extends BaseEntitySheet.Data<infer T> ? T : JournalEntry
+> extends BaseEntitySheet<D, O> {
+  constructor(object: O, options?: JournalSheet.Options);
+
```

Comment:
The constructor sets the protected property `_sheetMode`. This should be declared directly after the constructor. It needs to be of type `JournalSheet.SheetMode | null` because that's what `_inferDefaultMode` returns.